demos/custom-fields/field_turtle.js

```diff
@@ -310,14 +320,14 @@ CustomFields.FieldTurtle.prototype.renderEditor_ = function() {
 CustomFields.FieldTurtle.prototype.updateSize_ = function() {
   var size = this.movableGroup_.getBBox();
   if (this.borderRect_) {
-    this.borderRect_.setAttribute('width',
-      size.width + Blockly.BlockSvg.SEP_SPACE_X);
-    this.borderRect_.setAttribute('height', size.height + 9);
+    size.width += Blockly.Field.X_PADDING;
+    size.height += Blockly.Field.X_PADDING;
```

Comment:
Maybe Blockly.Field.Y_PADDING? or just Blockly.Field.PADDING?

Reply:
Field doesn't have Y_PADDING, but I changed this to a PADDING property on this field.